timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -86,37 +95,38 @@ class inheriting from ``AbstractTrainer`` which controls training of multiple mo
     quantiles : List[float]
         Alias for :attr:`quantile_levels`.
     ignore_time_index : bool, default = False
-        If True, AutoGluon-TimeSeries will ignore any date time indexes given in any dataset in train and test time,
-        and replace any input data indexes with dummy timestamps in second frequency. In this case, sktime models
-        will not activate any seasonality inference if not specified explicitly in ``hyperparameters``, and the
-        forecast output time indexes will be arbitrary values.
+        If True, the predictor will ignore the datetime indexes during both training and testing, and will replace
+        the data indexes with dummy timestamps in second frequency. In this case, the forecast output time indexes will
+        be arbitrary values, and seasonality will be turned off for local models.
     validation_splitter : Union[str, AbstractTimeSeriesSplitter], default = "last_window"
-        Strategy for splitting ``train_data`` into trainining and validation parts during
+        Strategy for splitting ``train_data`` into training and validation parts during
         :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit`. If ``tuning_data`` is passed to
         :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit`, validation_splitter is ignored. Possible choices:
 
         - ``"last_window"`` - use last ``prediction_length`` time steps of each time series for validation.
-        - ``"multi_window"`` - use last 3 non-overlapping windows of length ``prediction_length`` of each time series for validation.
-        - object of type :class:`~autogluon.timeseries.splitter.AbstractTimeSeriesSplitter` implementing a custom splitting strategy (for advanced users only).
+        - ``"multi_window"`` - use last 3 non-overlapping windows of length ``prediction_length`` of each time series
+            for validation.
+        - object of type :class:`~autogluon.timeseries.splitter.AbstractTimeSeriesSplitter` implementing a custom
+            splitting strategy (for advanced users only).
 
 
+    # TODO: Why is only target listed under Attributes and not, e.g., eval_metric? Do we even need to mention anything here?
     Attributes
     ----------
     target : str
-        Name of column in training/validation data that contains the target time-series value to be predicted. If
-        not specified explicitly during :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit`, this will default to
-        ``"target"``.
+        Name of column in training/validation data that contains the target time series value to be predicted.
```

Comment:
I see now that this should be "if not specified in the constructor," but why are we removing the explanation? This is an attribute (property) and does not really "default" to anything?

Reply:
To rephrase my question a bit: A `TimeSeriesPredictor` object has many other attributes (e.g., `eval_metric`, `quantile_levels`, `path`) that behave exactly like `target` - they are specified in the constructor and have some default values. Is there some reason why only `target` is mentioned under `Attributes`, but the others are not?

I feel like we can safely remove this, as we don't expect users to use these attributes in their code.

Reply:
Ah yes, fair. OK to remove.